metricbeat/docs/modules/kibana.asciidoc

```diff
@@ -3,10 +3,9 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-module-kibana]]
-== kibana Module
-
-This is the kibana Module.
+== Kibana Module
 
+This is the Kibana Module and only tracks the high level metrics. To monitor more Kibana metrics, use our {monitoringdoc}/xpack-monitoring.html[X-Pack monitoring] which is available under a free basic license.
```

Comment:
It might be worth adding a bit more clarification so users know what you mean by high-level metrics. So something like this:

The Kibana module tracks high-level metrics, such as the number of current connections, the number of requests, and the overall status of Kibana.

I'd keep it short to give users a general sense of what we mean by high-level.